### ISSUE TITLE
Delete scheduler for FixApplicantDobDataPathJob

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#6897]
Before we can remove the code that runs `FixApplicantDobDataPathJob`, we have to let the existing jobs run and remove the scheduler so no new jobs are scheduled. Once this is released and all deployments are on the release version with this change, we can proceed to the next step which is deleting the code that runs the job + tests. (I learned the hard way in https://github.com/civiform/civiform/issues/6284 that doing this all in one go results in an error).

**Comment by rockycodes** [comment]
Follow-up issue: https://github.com/civiform/civiform/issues/6898

**Comment by rockycodes** [comment]
Update: no TDD was made for this issue. Nick and I settled on a simple implementation where a job that is found in the db but not the registry is deleted from the db.
